pkg/errors/errors.go

```diff
@@ -284,6 +287,10 @@ func IsErrDeadlineExceeded(err error) bool {
 	return checkError(err, os.ErrDeadlineExceeded)
 }
 
+func IsWMIError(err error) bool {
+	return checkError(err, WMIError)
```

Comment:
This won't work. Also, you'll want to move "IsWMIError" to the WMI module (see my comments in the WMI module)